.github/actions/deploy/deploy.py

```
@@ -0,0 +1,1227 @@
+        deployment_mode = 'operator' if use_operator else 'direct'
+
+        if use_operator:
+            # Operator deployment
+            deployment_name = f"ds-pipeline-{self.args.dspa_name}"
+            service_account_name = f"ds-pipeline-{self.args.dspa_name}"
+        else:
+            # Direct deployment (multi-user or proxy)
+            deployment_name = 'ml-pipeline'
+            service_account_name = 'ml-pipeline'
+
+        print(f"📤 Outputting deployment metadata:")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)